src/assets/conclusion.js

```diff
@@ -0,0 +1,29 @@
+import dial from "./dial.jpg";
+import looking from "./looking.jpg";
+import whyad from "./whyamiseeingthisad.png";
+export default [
+  {
+    card: {
```

Comment:
Is there a reason why there's a `card` key? It seems like we could just remove it and shift the inside dictionary up one level.